zhaquirks/tuya/ts011f_plug.py

```diff
@@ -1170,6 +1170,65 @@ class Plug_v3(EnchantedDevice):
     }
 
 
+class Plug_v3_noGP(EnchantedDevice):
```

Comment:
Yeah, the Tuya plug class names are kind of all over the place already.
Generally, we don't even want underscores in class names, but we're doing this for other Tuya pugs already.

Maybe just capitalize the `N`, so use `Plug_v3_NoGP` instead.